emiliui.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="emiliui.css">
+    <title>Tado užduotis</title> 
+</head>
+<body>
+    <div class="container">
+        <fieldset>
+            <header class="header">
+                <div class="h1">
+                    <h1>Best <id class="blue">ToDo</id>  App</h1>
+                            <a href="https://www.facebook.com/"><i class="fa-brands fa-facebook"></i></a>
```

Comment:
For font-awesome icons to work, you'll have to include the font-awesome library into your <head>. 
Here's the link to add: https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css 
(Add it the way we add styles files)